docs/sources/mimir/operators-guide/deploy-grafana-mimir/jsonnet/migrate-from-microservices-to-read-write.md

```diff
@@ -0,0 +1,311 @@
+---
+description:
+  Learn how to migrate from a cluster deployed with as microservices to
+  a cluster in read-write mode.
+menuTitle: Migrate from microservices to read-write mode
+title: Migrate from microservices to read-write mode without downtime
+weight: 40
+---
+
+# Migrate from microservices to read-write mode without downtime
+
+> **Warning:**
+> Read-Write mode is considered experimental, as is migrating between the modes.
+
+At a high level the basic process of steps to complete is:
+
+1. Deploy read-write components along side microservices (they join the same ring).
+1. Switch over end points in your ingress.
```

Comment:
I need to revisit the structure of this doc because the high-level steps can have `Step 1:` etc. but there are many steps. I will come back to this tomorrow.

Reply:
The high level steps are meant to portray the general process. There are a number of specific items to complete in order to do what the process is describing. I'm open to suggestions.

Reply:
I will look at the structure in a subsequent PR. Best to iterate as you suggested in a DM. :)